### PR TITLE
Add SPM support

### PR DESCRIPTION
**Pull request by THISISDINOSAUR**
Adds swift package manager support, and changes some of the structure to make it more swift packagy/updates the readme

### CODE REVIEW

File: Sources/FindInPageIOSJSSupport/FindInPageIOSJSSupport.swift [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import Foundation
+
+public struct FindInPageIOSJSSupport {
+    public static var Bundle: Bundle = .module
```
Comment (brindy) [inline Sources/FindInPageIOSJSSupport/FindInPageIOSJSSupport.swift]: Isn't the name of this var wrong?  Shouldn't it be:

```suggestion
    public static var bundle: Bundle = .module
```

**Review by brindy — CHANGES_REQUESTED** [review]
Added a comment about var naming.

File: Sources/FindInPageIOSJSSupport/FindInPageIOSJSSupport.swift (hunk 1 above)
Comment (THISISDINOSAUR) [reply Sources/FindInPageIOSJSSupport/FindInPageIOSJSSupport.swift]: That's a good point, I used the content scope scripts one as a starting point which is wrong

**Review by brindy — APPROVED** [review]
LGTM

**Pull request merged**